doc/examples.xml

```diff
@@ -313,6 +313,51 @@ gap> GeneralisedPetersenGraph(IsMutableDigraph, 9, 4);
 </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="BishopsGraph">
+<ManSection>
+  <Oper Name="BishopsGraph" Arg="[filt, ]color, m, n"/>
+  <Returns>A digraph.</Returns>
+  <Description>
+    If <A>m</A> and <A>n</A> are positive integers and <A>color</A> is
+    either the string <C>"dark-square"</C> or the string <C>"light-square"</C>,
+    then this operation returns the <E>bishop's graph</E> for an <A>m</A> by
+    <A>n</A> board for a bishop of that square color.<P/>
+
+    From
+    <URL>https://mathworld.wolfram.com/BishopGraph.html</URL>:
+    <P/>
+
+    <Q> A bishop's graph is a graph formed from possible moves of a bishop chess
+    piece, which may make diagonal moves of any length on a chessboard (or any 
+    other board). To form the graph, each chessboard square is considered a vertex, 
```

Comment:
I'm not sure I understand the comment "any other board". I suppose you mean not necessarily 8x8 board? Maybe we could make it a bit more precise and say that any other board would again have black and white squares like chessboards etc :)